src/CharacterCard.js

```diff
@@ -11,10 +30,16 @@ class CharacterCard extends React.Component {
         <a href="/" className="header">
           {this.props.character.name}
         </a>
-        <div className="description">
+        <div className="meta">
           {this.props.character.title}
         </div>
+        <div className="description">
+          { this.state.display }
+        </div>
       </div>
+      <button className="ui teal button" onClick={() => this.getQuote(this.props.character.name)}>
+        Beer Me That Quote
```

Comment:
To make things a bit cleaner, I would destructure `this.props` to pull out the `name` out like so:
```
const {
  character: { name }
} = this.props;
```

Now you can just pass `name` into the getQuote function. Here's an article on this: https://wesbos.com/destructuring-objects

Reply:
I think you would need to do this outside of the return function, but inside the render() function

Reply:
```
const {
  this: {
     character: { name }
  }
} = props;
```